README.md

```diff
@@ -78,3 +78,11 @@ terminus(server, {
 
 server.listen(3000);
 ```
+
+## Windows support
```

Comment:
Maybe changing the name to “Limited Windows support”?, just to signal the content in the header too

Reply:
Done.